pages/levels/5.vue

```diff
@@ -0,0 +1,89 @@
+<script lang="ts">
+import { defineComponent } from "vue";
+
+const code = `    `;
+
+export default defineComponent({
+  data: () => {
+    return {
+      markdown: `What’s the deal with POSH? POSH stands for \'Plain Old Semantix HTML\'. It basically means you should use the right HTML elements for their intended purpose, like using a button instead of a div with a click function.
```

Comment:
c statt x
```suggestion
      markdown: `What’s the deal with POSH? POSH stands for \'Plain Old Semantic HTML\'. It basically means you should use the right HTML elements for their intended purpose, like using a button instead of a div with a click function.
```